apps/daffio/src/app/content/docs/api-docs/views/api-doc/api-doc-view.component.spec.ts

```diff
@@ -0,0 +1,32 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+import { RouterTestingModule } from '@angular/router/testing';
+import { NoopAnimationsModule } from '@angular/platform-browser/animations';
+
+import { ApiDocViewComponent } from './api-doc-view.component';
+import { DaffioDocsApiModule } from '../../api.module';
+
+describe('ApiDocViewComponent', () => {
+  let component: ApiDocViewComponent;
+  let fixture: ComponentFixture<ApiDocViewComponent>;
+
+  beforeEach(async(() => {
+    TestBed.configureTestingModule({
+      imports: [
+        DaffioDocsApiModule,
+        RouterTestingModule,
+        NoopAnimationsModule
```

Comment:
Why do we need this NoopAnimationsModule?

Reply:
I think it was previously there to absorb the 404 routing errors being thrown by the `daff-api-doc-container` (or whatever it is), but I just mocked it out instead and removed the NoopAnimationsModule